awsmp/cli.py

```diff
@@ -315,45 +319,46 @@ def ami_product_update_version(product_id, config):
 
 
 @public_offer.command("update-legal-terms")
-@click.option("--offer-id", required=True, prompt=True)
```

Comment:
do those CLI interface changes require a documentation update?

Reply:
yes, I should update document too! thanks for reminding that!